node/src/reactor.rs

```diff
@@ -52,8 +54,10 @@ use quanta::Clock;
 pub use queue_kind::QueueKind;
 use tokio::time::{Duration, Instant};
 
-/// Threshold for when an event is considered slow.
-const DISPATCH_EVENT_THRESHOLD: Duration = Duration::from_millis(1);
+/// Default threshold for when an event is considered slow.  Can be overridden by setting the env
+/// var `CL_EVENT_MAX_USEC=<MICROSECONDS>`.
+const DEFAULT_DISPATCH_EVENT_THRESHOLD: Duration = Duration::from_secs(1);
+const DISPATCH_EVENT_THRESHOLD_ENV_VAR: &str = "CL_EVENT_MAX_USEC";
```

Comment:
I'm not sure that `U` is needed, I think it's pretty common to just end it with the unit so `_SEC` should suffice.

Reply:
Ah - that's a bad sign.  The `U` was to indicate microseconds rather than units.  Maybe I should just go for the longer `_MICROSECS` or something?

Reply:
You could also do `uSEC`

Reply:
If it's really supposed to be microseconds then yes, `_MICROSECS` sounds better.